### PR TITLE
Fix Scan max channels

### PR DESCRIPTION
**Pull request by kapbh**

### CODE REVIEW

**Review by krish2718 — DISMISSED** [review]
The first commit log is too programmatic, it could use some re-wording.

File: subsys/net/l2/wifi/wifi_shell.c [hunk 1]
```diff
@@ -1676,7 +1676,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
...
 		    "[-s, --ssid : SSID to scan for. Can be provided multiple times\n"
 		    "[-m, --max_bss <val>] : Maximum BSSes to scan for. Range 1 - 65535\n"
-		    "[-c, --chans <Comma separated list of channel ranges>] : Channels to be scanned. The channels must be specified in the form band1:chan1,chan2_band2:chan3,..etc. band1, band2 must be valid band values and chan1, chan2, chan3 must be specified as a list of comma separated values where each value is either a single channel or a channel range specified as chan_start-chan_end. Each band channel set has to be separated by a _. For example, a valid channel specification can be 2:1,6-11,14_5:36,149-165,44\n"
+		    "[-c, --chans <Comma separated list of channel ranges>] : Channels to be scanned. The channels must be specified in the form band1:chan1,chan2_band2:chan3,..etc. band1, band2 must be valid band values and chan1, chan2, chan3 must be specified as a list of comma separated values where each value is either a single channel or a channel range specified as chan_start-chan_end. Each band channel set has to be separated by a _. For example, a valid channel specification can be 2:1,6_5:36\n"
```
Comment (jukkar) [inline subsys/net/l2/wifi/wifi_shell.c:1679]: No objections to this change, I am just wondering why checkpatch did not complain about the very long line.

**Review by jukkar — DISMISSED** [review]

File: subsys/net/l2/wifi/wifi_shell.c (hunk 1 above)
Comment (kapbh) [reply subsys/net/l2/wifi/wifi_shell.c:1679]: Maybe checkpatch ignores it. Do that need change to short lines?

Comment (krish2718) [reply subsys/net/l2/wifi/wifi_shell.c:1679]: String only long lines are ignored by checkpatch.

**Review by jukkar — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Review by krish2718 — APPROVED** [review]

**Pull request merged**